src/js/filter.js

```diff
@@ -1,43 +1,47 @@
 /* global document, window */
 
-const setUpFilter = (markerGroup, citiesToMarkers, data) => {
-  const filter = document.querySelector(".filter--scope");
-  const priorSelection = [
-    "Regional",
-    "Citywide",
-    "City Center",
-    "Transit Oriented",
-    "Main Street",
-  ];
+/**
+ * Helper function to iterate over every city and either remove it or add it,
+ * depending on the result of filterFn for that city.
+ *
+ * @param filterFn: a function of (string) => boolean. That is, a function that takes
+ *    one argument for the cityState name, e.g. 'Boston, MA', and returns true if it
+ *    should be displayed on the map.
+ */
+const changeSelectedMarkers = (markerGroup, citiesToMarkers, filterFn) => {
+  Object.entries(citiesToMarkers).forEach(([cityState, marker]) => {
+    if (filterFn(cityState)) {
+      marker.addTo(markerGroup);
+    } else {
+      marker.removeFrom(markerGroup);
+    }
+  });
+};
 
-  const onScopeFilterChange = (markerGroup, citiesToMarkers, data) => () => {
-    const selected = new Set(
-      [...document.querySelectorAll(".filter--scope :checked")].map(
-        (option) => option.value
-      )
-    );
-    Object.entries(citiesToMarkers).forEach(([cityState, marker]) => {
-      if (
-        data[cityState]["report_magnitude"]
-          .split(",")
-          .some((scope) => selected.has(scope))
-      ) {
-        marker.addTo(markerGroup);
-      } else {
-        marker.removeFrom(markerGroup);
-      }
-    });
-  };
+// The double => is "partial application".
+const onScopeFilterChange = (markerGroup, citiesToMarkers, data) => () => {
+  const selected = new Set(
+    [...document.querySelectorAll(".filter--scope :checked")].map(
+      (option) => option.value
+    )
+  );
+  changeSelectedMarkers(markerGroup, citiesToMarkers, (cityState) =>
+    data[cityState]["report_magnitude"]
+      .split(",")
+      .some((scope) => selected.has(scope))
+  );
+};
 
-  // makes all options selected
+const setUpFilter = (markerGroup, citiesToMarkers, data) => {
+  const filter = document.querySelector(".filter--scope");
+  // Pre-select all options.
```

Comment:
Btw, a lot of projects have a code style with comments that they should be a sentence with a period. Others don't care. I got in the habit of it though.